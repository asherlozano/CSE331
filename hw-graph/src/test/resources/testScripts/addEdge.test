# addEdge expected file
# tests if an edge can be added to a graph
# an edge can be used to connect nodes and create a path

# Create a graph
CreateGraph g1

# Create a node
AddNode g1 n1

# Create an edge
AddEdge g1 n1 n1 e1

# Print the nodes in the graph
ListNodes g1