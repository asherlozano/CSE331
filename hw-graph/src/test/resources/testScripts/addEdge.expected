# addEdge expected file
# tests if an edge can be added to a graph
# an edge can be used to connect nodes and create a path

# Create a graph
created a graph g1

# Create a node
added node n1 to g1

# Create an edge
added edge e1 from n1 to n1 in g1

# Print the nodes in the graph
g1 contains: n1